src/App.js

```diff
@@ -0,0 +1,33 @@
+import { BrowserRouter,Route,Routes } from 'react-router-dom';
+// import './App.css';
+import Login from './Pages/login';
+import Welcome from './Pages/welcome';
+
+function App() {
+  return (
+    <BrowserRouter >
```

Comment:
```suggestion
    <BrowserRouter>
```
You should also look at small errors like this. Packages like `ESLint` does help in production codebases. But it's good to have on things like these from the start.

Reply:
Fixed e97eae75b859e2f1f70c09d6abe315d63c384f20